### PR TITLE
Fully functional import nobugs

### PR DESCRIPTION
**Pull request by CristiArde**
no more bugs on import. fully functional with crypto and regular stocks

### CODE REVIEW

File: src/main/java/nitezh/ministock/MimeSendTask.java [hunk 1]
```diff
@@ -111,17 +111,23 @@ protected Void doInBackground(Void... voids) {
...
             Transport.send(mm);
 
+            /* Causes problem in testing
             Toast.makeText(context,
                     "Sending e-mail to " + toAddress, Toast.LENGTH_LONG).show();
+                    */
 
         } catch (MessagingException max) {
             max.printStackTrace();
+            /* Causes problem in testing
```
Comment (GurkomalSRao) [inline src/main/java/nitezh/ministock/MimeSendTask.java]: remove commented code

File: src/main/java/nitezh/ministock/activities/GlobalWidgetData.java [hunk 2]
```diff
@@ -28,7 +29,7 @@
 
     // Use in population of listview 
     public static List<WidgetRow> myStockList = new ArrayList<WidgetRow>();
-    public static List<String> importSymbols = new ArrayList<>();
+    public static HashMap<String,String> importSymbolsAndNames = new HashMap<>();
```
Comment (GurkomalSRao) [inline src/main/java/nitezh/ministock/activities/GlobalWidgetData.java]: importSymbols AndNames not used and should be removed

File: src/main/java/nitezh/ministock/activities/GlobalWidgetData.java [hunk 3]
```diff
@@ -38,19 +39,11 @@
         return myStockList;
     }
 
-    public List<String> getImportSymbols() {
-        return importSymbols;
-    }
-
-    public void setImportSymbols(List<String> symbols)
```
Comment (GurkomalSRao) [inline src/main/java/nitezh/ministock/activities/GlobalWidgetData.java:45]: getImportSymbols() method not used

File: src/main/java/nitezh/ministock/activities/PreferencesActivity.java [hunk 4]
```diff
@@ -43,8 +42,6 @@ of this software and associated documentation files (the "Software"), to deal
 import android.preference.Preference.OnPreferenceClickListener;
 import android.preference.PreferenceActivity;
```
Comment (GurkomalSRao) [inline src/main/java/nitezh/ministock/activities/PreferencesActivity.java:43]: alot of unnecessary imports. remove them all pls

**Review by GurkomalSRao — CHANGES_REQUESTED** [review]

**Review by Nick9500 — APPROVED** [review]

**Review by GurkomalSRao — APPROVED** [review]

**Pull request merged**